Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM golang:1.14-alpine3.11 as builder
 LABEL maintainer="cncf-falco-dev@lists.cncf.io"
 
 RUN apk add --no-cache make bash git
+RUN mkdir /var/lib/rpm/
```

Comment:
It does not seem to me that is fixing the issue:
```
$ docker run falcosecurity/event-generator:032fcc4c9c9211fe51a54545c4a4c46e258a6425 run syscall.WriteBelowRpmDatabase
INFO writing to /var/lib/rpm/created-by-event-generator  action=syscall.WriteBelowRpmDatabase
ERRO action error                                  action=syscall.WriteBelowRpmDatabase error="open /var/lib/rpm/created-by-event-generator: no such file or directory"
```

It should be moved to the last stage (below the `FROM alpine:3.11`) to be included in the image filesystem.